app/templates/main/map.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'layouts/base.html' %}
+{% block content %}
+<link rel="stylesheet" href="static/styles/iThing.css" type="text/css" />
+<link rel="stylesheet" href="static/styles/map.css" type="text/css" />
+<script src="static/js/jquery-ui.js"></script>
+<script src="static/js/jQDateRangeSlider-min.js"></script>
+<script src="static/js/initialize_map.js"></script> 
+<meta id="vehicle_ids" data-name="{{vehicle_ids}}">
```

Comment:
Hmm, I've never seen it done like this. Usually, I've seen jinja -> javascript done with the jinja `json` (or something like this) functions.

This won't matter though assuming you make the changes I described earlier.